uid2_client/identity_map_client.py

```diff
@@ -0,0 +1,40 @@
+import base64
+import datetime as dt
+from datetime import timezone
+
+from .identity_map_response import IdentityMapResponse
+
+from uid2_client import auth_headers, make_v2_request, post, parse_v2_response
+
+
+class IdentityMapClient:
+    """Client for interacting with UID Identity Map services
+
+        You will need to have the base URL of the endpoint and a client API key
+        and secret to consume web services.
+
+        Methods:
+            generate_identity_map: Generate identity map
+    """
+
+    def __init__(self, base_url, auth_key, secret_key):
+        """Create a new IdentityMapClient client.
+
+        Args:
+            base_url (str): base URL for all requests to UID services (e.g. 'https://prod.uidapi.com')
+            auth_key (str): authorization key for consuming the UID services
```

Comment:
This is called an API key, not an auth key https://unifiedid.com/docs/ref-info/glossary-uid
And the secret is `client secret` or `API secret`

Reply:
I updated it to `api_key` and `client_secret`